_plugins/gtn.rb

```diff
@@ -332,6 +332,13 @@ def convert_to_material_list(site, materials)
         if m.key?('name') && m.key?('topic')
           found = TopicFilter.fetch_tutorial_material(site, m['topic'], m['name'])
           Jekyll.logger.warn "Could not find material #{m['topic']}/#{m['name']} in the site data" if found.nil?
+
+          if m.key?('time_start')
+            found = found.merge('time_start' => m['time_start'])
```

Comment:
```suggestion
            found['time_start'] = m['time_start']
```

i think this works as well?